### ISSUE TITLE
Can't quit discord

### ISSUE THREAD

**Issue by piotrpalek** [snapcrafters/discord#17]
Hey, I'm trying to quit discord installed via snap and can't do it. I need to use `killall Discord` (sometimes twice because it restarts itself). I'm trying to quit it by right clicking the tray icon and selecting "quit".

Some more info:
`discord  0.0.4                    52    snapcrafters`
This is on Solus Linus

**Comment by Ads20000** [comment]
Sounds like #3 has resurfaced...

**Comment by aptanet** [comment]
I have the same problem, which I noticed when I had a notification after I had quit Discord. Similarly, when I tried to kill the Discord process it simply started Discord keeping the process number the same; second time around it did kill Discord.

This is running on Ubuntu 17.10 (stock with Gnome). Unfortunately there is no tray icon for Discord, so the only option to kill is on the menu at the top right, where choosing Quit simply appears to close the app, but leaves the three processes running:

paul      4074 18.5  1.4 1349768 118788 tty2   Sl+  21:21   0:01 /snap/discord/52/usr/share/discord/Discord
paul      4188  0.0  0.3 326712 28140 tty2     S+   21:21   0:00 /snap/discord/52/usr/share/discord/Discord --type=zygote --no-sandbox
paul      4247 50.3  2.7 1994876 221808 tty2   Sl+  21:21   0:03 /snap/discord/52/usr/share/discord/Discord --type=renderer --no-sandbox --primordial-pipe-token=1B8B9BBF12913B7B954DF51FD1F88B85 --lang=en-GB --app-pa

name:      discord
summary:   All-in-one voice and text chat for gamers
publisher: snapcrafters
contact:   https://github.com/snapcrafters/discord/issues
license:   unknown
description: |
  All-in-one voice and text chat for gamers that's free, secure, and
  works on both your desktop and phone.
snap-id: qHVefGEBezeuCeSfTND40uoUD6GRw8BO
commands:
  - discord
tracking:    stable
installed:   0.0.4 (52) 131MB -
refreshed:   2018-01-09 11:11:20 +0000 UTC
channels:               
  stable:    0.0.4 (52) 131MB -
  candidate: 0.0.4 (52) 131MB -
  beta:      0.0.4 (52) 131MB -
  edge:      0.0.4 (53) 127MB -


name:      core
summary:   snapd runtime environment
publisher: canonical
contact:   snappy-canonical-storeaccount@canonical.com
license:   unknown
description: |
  The core runtime environment for snapd
type:        core
snap-id:     99T7MUlRhtI3U0QFgl5mXXESAiSwt776
tracking:    stable
installed:   16-2.31.1 (4110) 85MB core
refreshed:   2018-02-20 17:07:19 +0000 UTC
channels:                                    
  stable:    16-2.31.1                (4110) 85MB -
  candidate: 16-2.31.1                (4110) 85MB -
  beta:      16-2.32~pre1             (4139) 85MB -
  edge:      16-2.31.1+git593.da27fef (4156) 85MB -
